src/guides/v2.3/release-notes/release-notes-2-3-5-open-source.md

```diff
@@ -390,6 +402,14 @@ We have fixed hundreds of issues in the Magento 2.3.5 core code.
 
 <!--- MC-22819-->
 
+*  We’ve improved the reliability of background `cron` execution. `cron:run` execution no longer causes an error on the application level, and `cron` jobs now rely on the lock framework.
```

Comment:
This explanation is pretty vague, is there more info about what is improved here? A lot of people are having strange cron-related problems, so more detailed explanation might be useful here.

Reply:
I'll follow up on this with Misha -- thanks for pointing this out. I would love to eliminate or at least  shed light on cron mysteries!